spatialmath/box.go

```diff
@@ -195,18 +217,25 @@ func (b *box) penetrationDepth(pt r3.Vector) float64 {
 // boxVsBoxCollision takes two boxes as arguments and returns a bool describing if they are in collision,
 // true == collision / false == no collision.
 func boxVsBoxCollision(a, b *box) bool {
-	positionDelta := PoseDelta(a.pose, b.pose).Point()
+	centerDist := b.pose.Point().Sub(a.pose.Point())
+
+	// check if there is a distance between bounding spheres to potentially exit early
+	if centerDist.Norm()-(a.boundingSphereR+b.boundingSphereR) > 0 {
```

Comment:
[nit] This should be greater than CollisionBuffer as well

Reply:
done